flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/plan/nodes/CommonPythonCalc.scala

```diff
@@ -54,6 +57,10 @@ trait CommonPythonCalc {
           val argPythonInfo = createPythonScalarFunctionInfo(pythonRexCall, inputNodes)
           inputs.append(argPythonInfo)
 
+        case literal: RexLiteral =>
+          inputs.append(
+            rexLiteralToPythonObject.invoke(null, literal.getValue3, literal.getTypeName))
```

Comment:
Should convert `literal.getValue3` to the actual type. For example, for bigint type, the type of `literal.getValue3` is `BigDecimal`, which is not correct. You can refer to `CodeGenerator#visitLiteral` for more information on how to do that.